<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContributorsPage</class>
 <widget class="QDialog" name="ContributorsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contributors</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="programmingTab">
      <attribute name="title">
       <string>Written by</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextBrowser" name="programmerText"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="translationsTab">
      <attribute name="title">
       <string>Translated by</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTextBrowser" name="translatorText"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="artworkTab">
      <attribute name="title">
       <string>Artwork by</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTextBrowser" name="artworkText"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
